api/types/mount/mount.go

```diff
@@ -18,7 +18,7 @@ const (
 	// TypeNamedPipe is the type for mounting Windows named pipes
 	TypeNamedPipe Type = "npipe"
 	// TypeCluster is the type for Swarm Cluster Volumes.
-	TypeCluster = "csi"
+	TypeCluster Type = "cluster"
```

Comment:
I _think_ the intent of @dperny was to change this from `csi` to `cluster` ?


There's other occurrences of CSI in the codebase though, and I wasn't sure which ones should be renamed and which ones not.